brave/ui/app/pages/keychains/restore-vault.js

```diff
@@ -1,6 +1,12 @@
-import RestoreVaultPage from '../../../../../ui/app/pages/keychains/restore-vault'
+const RestoreVaultPage = require('../../../../../ui/app/pages/keychains/restore-vault').default.WrappedComponent
```

Comment:
The issue here is what was being extended before was not a react class, but the entire connected component (https://github.com/brave/ethereum-remote-client/blob/master/ui/app/pages/keychains/restore-vault.js#L190)

Since the above MM doesn't have a separate container file to override, all we can do is extract the `WrappedComponent` which contains the class we want to extend, and then re-connect it.